platforms/nuttx/src/px4/common/hrt_ioctl.c

```diff
@@ -59,6 +59,21 @@ static sq_queue_t callout_queue;
 static sq_queue_t callout_freelist;
 static sq_queue_t callout_inflight;
 
+/* Check if entry is in list */
+
+bool entry_inlist(sq_queue_t *queue, sq_entry_t *item)
```

Comment:
maybe make it a static? it is only used in one place in here, compiler doesn't need to keep the function callable from outside

Reply:
Yes, should be static